lib/basedriver/commands/session.js

```diff
@@ -2,16 +2,26 @@ import _ from 'lodash';
 import log from '../logger';
 import { errors } from '../../mjsonwp';
 import UUID from 'uuid-js';
+import { processCapabilities } from '../capabilities';
 
 let commands = {};
 
-commands.createSession = async function (caps) {
+commands.createSession = async function (desiredCapabilities, requiredCaps, capabilities) {
```

Comment:
That's right. The only problem is that calls to `super.createSession` won't be passing in the 'capabilities' value to the base driver class (unless they did something like `super.createSession.apply(this, [...arguments])`